src/dcp/dcp.c

```diff
@@ -364,7 +364,21 @@ int main(int argc, char** argv)
         MPI_Finalize();
         return 1;
     }
-    
+   
+    /* check if source or destination is an HDF5 container, if that is
+     * the case then fork/exec h5repack */
+    rc = mfu_daos_hdf5_copy(mfu_src_file, mfu_dst_file, argpaths, daos_args);
```

Comment:
@dsikich We should only call this if either the source or dst is HDF5. Otherwise, we'll never get to the POSIX or DAOS copies